### ISSUE TITLE
A11y_ML .NET Model Builder_ Scenario_ Screen Reader: Narrator/NVDA is not reading any information when focus comes to any of the Scenarios

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1116]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev2; #NonBenchmark; #Visual Studio; #Screen Reader; #Narrator; #NVDA; #WCAG1.3.1;

### Environment Details:
Microsoft Visual Studio 2019 Preview 16.8.0 Version 6.0

### Screen Readers:
Narrator; NVDA

### Note:
Same issue is observed in "Next Steps" scenario as well

### Repro Steps:

1.Start Screen Reader
2.Open Visual Studio Enterprise 2019 Preview 
3. Create a new .Net Core Console App (C#) 
4. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
5. Verify how screen readers are reading the scenarios

### Actual Result:
Narrator/NVDA is not any information when focus comes to any of the Scenarios

### Expected Result:
Narrator/NVDA should read the name of the scenario properly

### User Impact:
Screen Reader users will not get to know about the scenario if screen readers doesn't read any information about them

### Attachment for Reference:
[1116_ML .NET Model Builder_ Narrator is not reading any info when focus is on scenario.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5603793/1116_ML.NET.Model.Builder_.Narrator.is.not.reading.any.info.when.focus.is.on.scenario.pptx)

[1116_ML .NET Model Builder_ NVDA is not reading any info when focus is on scenario.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5606272/1116_ML.NET.Model.Builder_.NVDA.is.not.reading.any.info.when.focus.is.on.scenario.pptx)

**Comment by beccamc** [comment]
Build to test here: https://devdiv.visualstudio.com/_apis/resources/Containers/7181026/VSSetup?itemPath=VSSetup%2FMicrosoft.ML.ModelBuilder.vsix.

Download and install the vsix on your Visual studio instance. This build can be used to test all the "Ready to review" bugs (should be all)

**Comment by poojapatil36** [comment]
Verified the issue in below version.
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.4.4
Issue is fixed, please find below attachment.
Github tag:; #Closed;
![1116_A11y_ML  NET Model Builder_ Scenario_ Screen Reader (2)](https://user-images.githubusercontent.com/97668775/218064899-ba4ab2a4-fbfc-431d-8593-fda381f952da.png)
![1116_A11y_ML  NET Model Builder_ Scenario_ Screen Reader](https://user-images.githubusercontent.com/97668775/218064911-29b4aa9e-3b00-41f7-9933-0a4fba876073.png)
